mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/matmul.mlir

```diff
@@ -0,0 +1,70 @@
+// RUN: mlir-opt %s -test-transform-dialect-interpreter -test-transform-dialect-erase-schedule \
+// RUN:   -one-shot-bufferize -func-bufferize -cse -canonicalize -convert-vector-to-scf -arm-sve-legalize-vector-storage \
+// RUN:   -convert-vector-to-llvm="enable-arm-sve" -test-lower-to-llvm | \
+// RUN: %mcr_aarch64_cmd -e=entry -entry-point-result=void --march=aarch64 --mattr="+sve" -shared-libs=%mlir_runner_utils,%mlir_c_runner_utils | \
+// RUN: FileCheck %s
+
+func.func @entry() {
+  %c1 = arith.constant 1 : index
+  %c2 = arith.constant 2 : index
+  %c4 = arith.constant 4 : index
+  %c0 = arith.constant 0 : index
+  %step = arith.constant 1 : index
+  %c0_f32 = arith.constant 0.0 : f32
+
+  %vscale = vector.vscale
+  %vl_fp = arith.muli %c4, %vscale : index
+  %A_alloc = bufferization.alloc_tensor(%c2, %c1) : tensor<?x?xf32>
+  %B_alloc = bufferization.alloc_tensor(%c1, %vl_fp) : tensor<?x?xf32>
+  %C_alloc = bufferization.alloc_tensor(%c2, %vl_fp) : tensor<?x?xf32>
+
+  %pi = arith.constant  3.14 : f32
+  %A = linalg.fill ins(%pi : f32) outs(%A_alloc : tensor<?x?xf32>) -> tensor<?x?xf32>
+  %B = linalg.fill ins(%pi : f32) outs(%B_alloc : tensor<?x?xf32>) -> tensor<?x?xf32>
+  %C_in = linalg.fill ins(%c0_f32 : f32) outs(%C_alloc : tensor<?x?xf32>) -> tensor<?x?xf32>
+
+  %C_out = linalg.matmul ins(%A, %B: tensor<?x?xf32>, tensor<?x?xf32>) outs(%C_in: tensor<?x?xf32>) -> tensor<?x?xf32>
+
+  // CHECK-LABEL: SVE: START OF TEST OUTPUT
+  vector.print str "SVE: START OF TEST OUTPUT"
+
+  // There are at least 4 x f32 elements in every SVE vector, i.e. 
+  //    * %vscale >= 1. 
+  // Hence, when checking the outupt there will always be at least 4 elements
+  // in every row. For implementations with wider vectors, you should see more
+  // elements being printed.
+  // CHECK: [9.8596,   9.8596,   9.8596,   9.8596
```

Comment:
```suggestion
  // CHECK-NEXT: [9.8596,   9.8596,   9.8596,   9.8596
```

Reply:
The actual output is:
```bash
SVE: START OF TEST OUTPUT
Unranked Memref base@ = 0x5500237580 rank = 2 offset = 0 sizes = [2, 16] strides = [16, 1] data =
[[9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596],
 [9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596,   9.8596]]
SVE: END OF TEST OUTPUT
```
Not sure it's worth checking the 2nd line?

Reply:
Currently it's doing:
```
// CHECK-LABEL: 
// CHECK:
// CHECK-NEXT:
```
But you could do:
```
// CHECK-LABEL: 
// CHECK-NEXT:
// CHECK-NEXT:
```
And that'd work too, and be nice an aligned.

Reply:
👍🏻 Updated, will land shortly.